tests/codemods/test_combine_startswith_endswith.py

```diff
@@ -37,4 +36,3 @@ def test_combine(self, tmpdir, func):
     )
     def test_no_change(self, tmpdir, code):
         self.run_and_assert(tmpdir, code, code)
```

Comment:
I'm not seeing num_changes=0 being passed so just want to make sure that removal of `assert len(self.file_context.codemod_changes) == 0` is indeed covered in new tests refactor

Reply:
Yes it's covered. The case where `expected == result` explicitly checks for that condition.